### ISSUE TITLE
Provide a generic contig for each species

### ISSUE THREAD

**Issue by apragsdale** [popsim-consortium/stdpopsim#668]
Currently, for each species, the only contigs available are the named chromosomes, which come with their set length, average recombination rate, etc. One usage of stdpopsim that I think will be common is to simulate under a demographic model with constant recombination rate for a given length (say 1 Mb). If I want to do this currently, I'd need to do something like
```
species = stdpopsim.get_species("HomSap")
contig = species.get_contig("chr22")
L = contig.recombination_map.get_sequence_length()
frac = 1e6 / L
contig = species.get_contig("chr22", length_multiplier=frac)
```
This contig then is the desired length, with the mutation and recombination rate for human chromosome 22. To me, this is a bit cumbersome just to get a contig of a given length.

A simple solution could be to allow a "generic" contig, where the length is directly specified:
```
species = stdpopsim.get_species("HomSap")
contig = species.get_contig("generic", length=1e6)
```
and this generic contig has the genome-wide average mutation and recombination rate for HomSap by default.

I think this would be a simple and useful addition to the API. Thoughts?

(This came out of discussions on masking simulated data that started here: #664)

**Comment by jeromekelleher** [comment]
I like it.  What if we did
```python
species = stdpopsim.get_species("HomSap")
contig = species.get_contig(length=1e6)
```

That is, we make the first arg an optional named arg.

**Comment by grahamgower** [comment]
Closed by #669.
